app/src/androidTest/java/com/duckduckgo/app/statistics/api/RxPixelSenderTest.kt

```diff
@@ -40,6 +43,17 @@ import org.junit.Test
 import org.mockito.Mock
 import java.util.concurrent.TimeoutException
 
+@Database(
+    entities = [PixelEntity::class],
+    version = 1
+)
+@TypeConverters(
+    QueryParamsTypeConverter::class
+)
+abstract class TestDatabase : RoomDatabase() {
+    abstract fun pixelDao(): PendingPixelDao
+}
+
```

Comment:
Created the `TestDatabase` to break the circular dependency with the main `app` module. Also, we don't need the full DB, just the DAO to perform the test

Reply:
as discussed, we can get rid of this